cylc/flow/tests/test_platform_lookup.py

```diff
@@ -110,10 +114,12 @@
                 "name": "hpc1-bg"
             },
         ),
-        (PLATFORMS_WITH_RE, "hpc2", {"login hosts": "hpc3", "name": "hpc2"}),
+        (PLATFORMS_WITH_RE, "hpc2", {"remote hosts": "hpc3", "name": "hpc2"}),
```

Comment:
I've just re-read this: This looks like we're testing against it behaving correctly but unpleasantly if the site admin messes up...